src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -34,6 +34,7 @@ const Body = BaseFormWithPolling.extend(
       BaseFormWithPolling.prototype.initialize.apply(this, arguments);
       this.listenTo(this.model, 'error', this.onPollingFail);
       doChallenge(this, IDENTIFIER_FLOW);
+      this.options.settings.set('isParallelPollCallsAllowed', true);
```

Comment:
what does this mean for poll requests ? `isParallelPollCallsAllowed`

Reply:
This code was removed